pkg/windows/windows.go

```diff
@@ -267,7 +279,7 @@ type Windows interface {
 	// Bootstrap prepares the Windows instance and runs the WICD bootstrap command
 	Bootstrap(context.Context, string, string, string) error
 	// ConfigureWICD ensures that the Windows Instance Config Daemon is running on the node
-	ConfigureWICD(string, string) error
+	ConfigureWICD(string, string, *WICDCertConfig) error
```

Comment:
Is this interface change necessary? What arguments would change between calls to ConfigureWICD?